packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -632,6 +632,7 @@ const TRANSFORMS = {
     const vdi = {
       type: 'VDI',
 
+      cbt_enabled: obj.cbt_enabled,
```

Comment:
`camelCase` for the property name?

Reply:
I would like to keep the xapi properties untouched if possible